API-strategie-modulen/API-strategie-mod-geo/request-response.md

```diff
@@ -1,10 +1,5 @@
 # Request and response
-
-Requesting resources and getting results back is an important part of any API. This is no different for REST APIs that handle geospatial data. There are, however, some specific aspects when dealing with geospatial data in REST APIs. The most important aspects are described in this chapter: 
-- how to supply a spatial filter in the call (request)
-- how to encode geometries in the response
-
-Note that when requesting information, for example about cadastral parcels, users do not necessarily require the geometry, even if they used a spatial filter. A name or parcel ID may be sufficient.
+REST APIs for handling geospatial features may provide spatial filtering. There is a distinction between retrieving geometries in the result (response) and supplying a spatial filter in the call (request). When requesting information, for example about cadastral parcels, users do not necessarily require the geometry. A name or parcel ID may be sufficient.
```

Comment:
Ik snap niet goed wat het doel is van dit hoofdstuk. De uitdagingen van geometrie in een REST API behandelen is goed, het is alleen opvallend dat bij request specifiek de spatial filtering genoemd wordt.

In het CRS hoofdstuk worden verschillende scenario's genoemd, waaronder een aantal waar geometrie in het verzoek zit. Zou dit hoofdstuk hier iets over moeten zeggen? 

De tekst lijkt nu vooral te gaan over het opvragen van geometrie via een REST API, maar objecten muteren zou toch ook moeten kunnen?

Reply:
Goed punt, het was nogal onduidelijk. Ik heb de tekst wat algemener gemaakt en duidelijk gemaakt dat create/update andere voorbeelden zijn waarbij geometrie in het request zit. 

Hiermee heb ik overigens ook meteen #486 meegenomen die hier over ging.

Kun je nog een keer reviewen @sweco-nlgemo ?